docs/api.md

```diff
@@ -102,7 +102,7 @@ mainly, but not only, provisioning details.
 
   | **hardwareProfile** | **Root Device** |
   |---------------------|-----------------|
-  | `unknown`           | /dev/sda        |
+  | `unknown`           | *none*          |
```

Comment:
Should we document what happens in this case?

Reply:
I added some text below the table.